### PR TITLE
CASMUSER-3063: Add topic of repurposed compute nodes as UANs

### PR DESCRIPTION
**Pull request by keopp-hpe**
# Description

This PR adds a reference to the UAN docs for repurposing compute nodes as UANs.

# Checklist Before Merging

- [NA] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [NA] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Repurpose_Compute_as_UAN.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Repurpose a Compute Node as a UAN
+
+It is possible to repurpose a compute node to be used as a User Access Node (UAN).  This is typically done when the processor type of the compute node is not yet available in a UAN server.
```
Comment (mharding-hpe) [inline operations/node_management/Repurpose_Compute_as_UAN.md:3]:
```suggestion
It is possible to repurpose a compute node to be used as a User Access Node (UAN). This is typically done when the processor type of the compute node is not yet available in a UAN server.
```

File: operations/node_management/Repurpose_Compute_as_UAN.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Please see the ***Repurposing a Compute Node as a UAN*** section of the *HPE Cray User Access Node (UAN) Software Administration Guide (S-8033)* for further information.
```
Comment (mharding-hpe) [inline operations/node_management/Repurpose_Compute_as_UAN.md:5]:
```suggestion
See the **Repurposing a Compute Node as a UAN** section of the *HPE Cray User Access Node (UAN) Software Administration Guide (S-8033)* for further information.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Comment by mharding-hpe** [comment]
Does this need a 1.3 backport?

**Comment by keopp-hpe** [comment]
@mharding-hpe , yes, this will need to be in 1.3 as the UAN docs supporting the repurposed compute nodes is in the UAN 2.5 which will go with 1.3.

File: operations/node_management/Repurpose_Compute_as_UAN.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Repurpose a Compute Node as a UAN
+
+It is possible to repurpose a compute node to be used as a User Access Node (UAN). This is typically done when the processor type of the compute node is not yet available in a UAN server.
+
+See the **Repurposing a Compute Node as a UAN** section of the *HPE Cray User Access Node (UAN) Software Administration Guide (S-8033)* for further information.
```
Comment (GunashekarKM) [inline operations/node_management/Repurpose_Compute_as_UAN.md:5]:
```suggestion
For more information, see the **Repurposing a Compute Node as a UAN** section of the *HPE Cray User Access Node (UAN) Software Administration Guide (S-8033)*.
```

File: operations/node_management/Repurpose_Compute_as_UAN.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Repurpose a Compute Node as a UAN
+
+It is possible to repurpose a compute node to be used as a User Access Node (UAN). This is typically done when the processor type of the compute node is not yet available in a UAN server.
+
+For more information, see the **Repurposing a Compute Node as a UAN** section of the *HPE Cray User Access Node (UAN) Software Administration Guide (S-8033)*.
```
Comment (mharding-hpe) [inline operations/node_management/Repurpose_Compute_as_UAN.md:5]:
```suggestion
For more information, see the **Repurposing a Compute Node as a UAN** section of the *HPE Cray User Access Node (UAN) Software Administration Guide (`S-8033`)*.
```

**Review by mharding-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**